style_exceptions/not_a_binary_url_prefix_allowlist.json

```diff
@@ -2,6 +2,7 @@
   "bittwist",
   "cspice",
   "flashrom",
+  "mlx",
```

Comment:
This was triggered for [`metal-lib`](https://developer.apple.com/metal/cpp/files/metal-cpp_macOS15_iOS18-beta.zip) which was part of our previous bottles due to setup.py running CMake. 

From general scan through, didn't see any binaries:
```console
❯ file --brief **/* | sort -u
C++ source text, ASCII text
C++ source text, ASCII text, with very long lines (304)
C++ source text, ASCII text, with very long lines (349)
C++ source text, ASCII text, with very long lines (367)
C++ source text, ASCII text, with very long lines (388)
C++ source text, ASCII text, with very long lines (397)
C++ source text, ASCII text, with very long lines (524)
Python script text executable, ASCII text
Unicode text, UTF-8 text
c program text, ASCII text
c program text, Unicode text, UTF-8 text, with very long lines (454)
directory
```